media-video/mjpg-streamer/files/mjpg-streamer.initd

```diff
@@ -34,8 +35,8 @@ start() {
 		--user "${MJPG_STREAMER_USER}" \
 		--group "${MJPG_STREAMER_GROUP}" -w 100 -b -m \
 		--pidfile "${MJPG_STREAMER_PIDFILE}" \
-		-- -i "/usr/lib/input_${INPUT_PLUGIN}.so ${INPUT_PLUGIN_OPTS}" \
-		-o "/usr/lib/output_${OUTPUT_PLUGIN}.so ${OUTPUT_PLUGIN_OPTS}"
+		-- -i "/usr/@LIBDIR@/input_${INPUT_PLUGIN}.so ${INPUT_PLUGIN_OPTS}" \
+		-o "/usr/@LIBDIR@/output_${OUTPUT_PLUGIN}.so ${OUTPUT_PLUGIN_OPTS}"
```

Comment:
wait, wait, this ebuild installs these plugins straight to /usr/lib? I think these should be installed in a subdir.

Reply:
yes, straight to /usr/lib. Will try to make a new directory mjpg-streamer and place plugins in there. But I wonder whether main binary (mjpg_streamer) will find those. Will try to figure this out.

Reply:
Is it ok to install plugins into **/usr/$(get_libdir)/mjpg-streamer/lib64/<plugin_name>.so** ?
'coz by default doins.so is appending that latter **lib64** directory entry. Apart from that it can be done like that and also changing entries in init.d file properly